client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/index.js

```diff
@@ -15,8 +15,8 @@ import Card from 'components/card';
 import ExtendedHeader from 'woocommerce/components/extended-header';
 import CarrierAccountListItem from './list-item';
 
-const CarrierAccounts = ({ translate, carriers = [] }) => {
-	if ( carriers.length === 0 ) {
+const CarrierAccounts = ({ translate, accounts = [] }) => {
```

Comment:
Just renames on the prop

Reply:
:information_source: Missing spaces inside parens.

Reply:
Doesn't seem to be enforced on `[]` anywhere else.